presentation/src/main/java/com/yapp/presentation/ui/member/todaysession/TodaySession.kt

```diff
@@ -24,11 +24,12 @@ import com.yapp.common.yds.YDSAppBar
 @Composable
 fun TodaySession(
     modifier: Modifier = Modifier,
+    navigateToMemberSettingScreen: () -> Unit,
 ) {
     Scaffold(
         topBar = {
             YDSAppBar(
-                onClickSettings = {}
+                onClickSettings = { navigateToMemberSettingScreen.invoke() }
```

Comment:
```suggestion
                onClickSettings = { navigateToMemberSettingScreen() }
```
invoke 는 생략하는게 어떨까용?

Reply:
invoke 안 붙이면 warning 뜨더라고요..!